src/Symfony/Bundle/WebProfilerBundle/Resources/config/profiler.php

```diff
@@ -0,0 +1,81 @@
+<?php
+
+/*
+ * This file is part of the Symfony package.
+ *
+ * (c) Fabien Potencier <fabien@symfony.com>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Symfony\Component\DependencyInjection\Loader\Configurator;
+
+use Symfony\Bundle\WebProfilerBundle\Controller\ExceptionPanelController;
+use Symfony\Bundle\WebProfilerBundle\Controller\ProfilerController;
+use Symfony\Bundle\WebProfilerBundle\Controller\RouterController;
+use Symfony\Bundle\WebProfilerBundle\Csp\ContentSecurityPolicyHandler;
+use Symfony\Bundle\WebProfilerBundle\Csp\NonceGenerator;
+use Symfony\Bundle\WebProfilerBundle\Twig\WebProfilerExtension;
+use Symfony\Component\HttpKernel\Debug\FileLinkFormatter;
+use Symfony\Component\VarDumper\Dumper\HtmlDumper;
+
+return static function (ContainerConfigurator $container) {
+    $container->services()
+
+        ->set('web_profiler.controller.profiler', ProfilerController::class)
+            ->public()
+            ->args([
+                service('router')->nullOnInvalid(),
+                service('profiler')->nullOnInvalid(),
+                service('twig'),
+                param('data_collector.templates'),
+                service('web_profiler.csp.handler'),
+                param('kernel.project_dir'),
+            ])
+
+        ->set('web_profiler.controller.router', RouterController::class)
+            ->public()
+            ->args([
+                service('profiler')->nullOnInvalid(),
+                service('twig'),
+                service('router')->nullOnInvalid(),
+            ])
+
+        ->set('web_profiler.controller.exception_panel', ExceptionPanelController::class)
+            ->public()
+            ->args([
+                service('error_handler.error_renderer.html'),
+                service('profiler')->nullOnInvalid(),
+            ])
+
+        ->set('web_profiler.csp.handler', ContentSecurityPolicyHandler::class)
+            ->args([
+                inline_service(NonceGenerator::class),
+            ])
+
+        ->set('twig.extension.webprofiler', WebProfilerExtension::class)
+            ->tag('twig.extension')
+            ->args([
+                inline_service(HtmlDumper::class)
+                    ->args([null, param('kernel.charset'), HtmlDumper::DUMP_LIGHT_ARRAY])
+                    ->call('setDisplayOptions', [['maxStringLength' => 4096, 'fileLinkFormat' => service('debug.file_link_formatter')]]),
+            ])
+
+        ->set('debug.file_link_formatter', FileLinkFormatter::class)
+            ->args([
+                param('debug.file_link_format'),
+                service('request_stack')->ignoreOnInvalid(),
+                param('kernel.project_dir'),
+                '/_profiler/open?file=%%f&amp;line=%%l#line%%l',
```

Comment:
Shouldn't `&amp;` become `&`?

Reply:
i think so too as `&amp;` was just for escaping in xml

Reply:
Both places are fixed.